converters/fromZigbee.js

```diff
@@ -342,7 +342,10 @@ const converters = {
         type: ['attributeReport', 'readResponse'],
         convert: (model, msg, publish, options, meta) => {
             if (msg.data.hasOwnProperty('lockState')) {
-                return {state: msg.data.lockState == 2 ? 'UNLOCK' : 'LOCK'};
+                return {
+                    state: msg.data.lockState == 1 ? 'LOCK' : 'UNLOCK',
+                    lockState: msg.data['lockState'],
```

Comment:
What is the added value of the lockState?

For reference: 

![image](https://user-images.githubusercontent.com/2892853/82759107-5e1c2680-9deb-11ea-9671-30645fb85ef5.png)

Reply:
As mentioned in the table, the value `lockState` indicates if the status of the lock and it gives more information than the binary lock/unlock status as the value 0x00 and 0x02 are both unlock, but the value 0x00 indicates the lock is not fully locked (i.e. if some error happen while trying to lock)

Reply:
Thanks, changed this to `action` and added a friendly text here so that it doesn't get cached.